src/MSBuild/XMake.cs

```diff
@@ -1920,10 +1926,15 @@ bool allowEmptyParameters
         private const string autoResponseFileName = "MSBuild.rsp";
 
         /// <summary>
-        /// THe name of an auto-response file to search for in the project directory and above.
+        /// The name of an auto-response file to search for in the project directory and above.
         /// </summary>
         private const string directoryResponseFileName = "Directory.Build.rsp";
 
+        /// <summary>
+        /// String replacement pattern to support paths in response files.
+        /// </summary>
+        private const string responseFilePathReplacement = "%MSBuildThisFileDirectory%";
```

Comment:
Is it possible to use the constant in `ReservedPropertyNames`?  Its internal and I don't think MSBuild.exe has access at the moment.

Reply:
It doesn't have InternalsVisibleTo for msbuild.exe, so I'm not sure it's worth it.